src/Symfony/EventListener/ReadListener.php

```diff
@@ -50,6 +51,7 @@ public function __construct(
         ?ResourceMetadataCollectionFactoryInterface $resourceMetadataCollectionFactory = null,
         private readonly ?SerializerContextBuilderInterface $serializerContextBuilder = null,
         LegacyUriVariablesConverterInterface|UriVariablesConverterInterface|null $uriVariablesConverter = null,
+        private readonly LoggerInterface $logger
```

Comment:
```suggestion
        private readonly ?LoggerInterface $logger = null,
```

(To prevent a BC break).

Reply:
sure, done.